watertap/unit_models/gac.py

```diff
@@ -1058,52 +1075,186 @@ def initialize_build(
+                    prop_name
+                ):
+                    var_dict[f"{prop_str} of {j} @ process feed inlet"] = getattr(
+                        self.process_flow.properties_in[time_point], prop_name
+                    )["Liq", j]
+                if self.process_flow.properties_out[time_point].is_property_constructed(
+                    prop_name
+                ):
+                    var_dict[f"{prop_str} of {j} @ process feed outlet"] = getattr(
+                        self.process_flow.properties_out[time_point], prop_name
+                    )["Liq", j]
+                if self.adsorbed_contam[time_point].is_property_constructed(prop_name):
+                    var_dict[f"{prop_str} of {j} @ process feed outlet"] = getattr(
+                        self.adsorbed_contam[time_point], prop_name
+                    )["Liq", j]
+
+        # loop throuhg desired state block properties indexed by [phase]
+        phase_prop_dict = {
+            "flow_vol_phase": "Volumetric flow rate",
+        }
+        for prop_name, prop_str in phase_prop_dict.items():
+            if self.process_flow.properties_in[time_point].is_property_constructed(
+                prop_name
+            ):
+                var_dict[f"{prop_str} @ process feed inlet"] = getattr(
+                    self.process_flow.properties_in[time_point], prop_name
+                )["Liq"]
+            if self.process_flow.properties_out[time_point].is_property_constructed(
+                prop_name
+            ):
+                var_dict[f"{prop_str} @ process feed outlet"] = getattr(
+                    self.process_flow.properties_out[time_point], prop_name
+                )["Liq"]
+            if self.adsorbed_contam[time_point].is_property_constructed(prop_name):
+                var_dict[f"{prop_str} @ process feed outlet"] = getattr(
+                    self.adsorbed_contam[time_point], prop_name
+                )["Liq"]
+
+        return {"vars": var_dict}
+
+    # ---------------------------------------------------------------------
+    def _get_stream_table_contents(self, time_point=0):
+        return create_stream_table_dataframe(
+            {
+                "Process Inlet": self.inlet,
+                "Process Outlet": self.outlet,
+                "Adsorbed Contaminant Outlet": self.adsorbed,
+            },
+            time_point=time_point,
+        )
 
     # ---------------------------------------------------------------------
     def calculate_scaling_factors(self):
         super().calculate_scaling_factors()
 
-        # overwrite default scaling
-        for j in self.config.property_package.solute_set:
-            sf = 100 * iscale.get_scaling_factor(
+        # scale based on molar flow traditionally provided by user for building flowsheets
+        for j in self.config.target_species:
+            sf_solute = iscale.get_scaling_factor(
+                self.process_flow.properties_in[0].flow_mol_phase_comp["Liq", j]
```

Comment:
We might want to add a warning and default here when the user does not provide the scaling factor.